90metalmdsquash/metal-genrules.sh

```diff
@@ -40,18 +40,22 @@ command -v wait_for_dev > /dev/null 2>&1 || . /lib/dracut-lib.sh
 command -v metal_die > /dev/null 2>&1 || . /lib/metal-lib.sh
 
 # Load and execute with desired URL driver.
-case "${metal_server:-}" in
-    file:*|http:*|https:*)
+IFS=':' read -ra ADDR <<< "$metal_server"
```

Comment:
`ADDR` is not used anywhere?

Also, while you're here, you should fix the nested quotes on line 66... where you're effectively un-quoting the variable.

Reply:
Actually, come to think of it this IFS line is really old and I think it was never used. In this change I had just this code block without much inspection.

Will cleanup.